src/js/main.js

```diff
@@ -0,0 +1,81 @@
+export const DiscourseRADParser = () => {
+  const radTabsElements = document.querySelectorAll('.rad-tabs-element');
+
+  radTabsElements.forEach((radTabsElement) => {
+    setDefaults(radTabsElement);
+    setupDropdowns(radTabsElement);
+  });
+
+  toggleElements();
+}
+
+const setDefaults = radTabsElement => {
+  const dropdowns = radTabsElement.querySelectorAll('select');
+
+  dropdowns.forEach((dropdown) => {
+    const name = dropdown.name;
+
+    const options = dropdown.querySelectorAll('option');
+    let defaultWasSet = false;
+    options.forEach((option) => {
+      if (localStorage.getItem(name) === option.value) {
+        defaultWasSet = true;
+        option.setAttribute('selected', 'selected');
+      }
+    });
+
+    if (!defaultWasSet) {
+      const firstOption = dropdown.firstElementChild;
+      firstOption.setAttribute('selected', 'selected');
+
+      if (localStorage.getItem(name) === null) {
+        localStorage.setItem(name, firstOption.value);
+      }
+    }
+  });
+}
+
+const setupDropdowns = radTabsElement => {
+  const dropdowns = radTabsElement.querySelectorAll('select');
+
+  dropdowns.forEach(function (dropdown) {
```

Comment:
You are mixing functions and arrow functions. Would be easier to read if it was consistent

Reply:
I replaced everything with arrow functions.